refactoring "info" and its links

mbaudis opened the pull request:
This PR addresses part of the "deep link to the framework" problems. It:
* moves the framework "info" to a separate file
* adds another info.yaml / .json to the models in "common" and changes refereces from model schemas to this local version

Both steps are complementary; if also going with the separate version in the model we might consider this for e.g. ontologyTerm, too (already a separate file in the framework).

mbaudis committed:
refactoring "info" and its links

This PR addresses part of the "deep link to the framework" problems. It:
* moves the framework "info" to a separate file
* adds another info.yaml / .json to the models in "common" and changes refereces from model schemas to this local version

Both steps are complementary; if also going with the separate version in the model we might consider this for e.g. ontologyTerm, too (already a separate file in the framework).

costero-e reviewed (approved):
I think this way will be cleaner for validators. Only a typo in individuals/defaultSchema.json, where an ontology is shown with closing parenthesis, line 15:  ```* female: NCIT:C16576)``` should be ```* female: NCIT:C16576```

mbaudis committed:
fix typo in example

Thanks @costero-e for https://github.com/ga4gh-beacon/beacon-v2/pull/184#pullrequestreview-2454894933 ...

mbaudis commented:
Thanks @costero-e; please check with @jrambla, or we do a merging session :-)

costero-e committed:
Merge branch 'schema-urgent-fixes' into refactor-deep-framework-links
